app/src/org/commcare/android/logging/ReportingUtils.java

```diff
@@ -158,4 +159,17 @@ public static String getPersonalID() {
         }
         return null;
     }
+
+    public static Boolean getIsDemoUser() {
```

Comment:
nit: rename to `isPersonalIdDemoUser` as CommCare has concept of demo users as well for non Personal ID accounts.